src/main/java/org/jenkinsci/deprecatedusage/DeprecatedUnusedApiReport.java

```diff
@@ -0,0 +1,141 @@
+package org.jenkinsci.deprecatedusage;
+
+import org.json.JSONObject;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.Writer;
+import java.util.List;
+import java.util.SortedSet;
+import java.util.TreeSet;
+
+public class DeprecatedUnusedApiReport extends Report {
+    public DeprecatedUnusedApiReport(DeprecatedApi api, List<DeprecatedUsage> usages, File outputDir, String reportName) {
+        super(api, usages, outputDir, reportName);
+    }
+
+    protected void generateHtmlReport(Writer writer) throws IOException {
+        writer.append("<h1>Unused Deprecated APIs</h1>");
+
+        {
+            writer.append("<div class='classes'><h2>Classes</h2><ul>\n");
```

Comment:
I thought about that and I'm not sure what the goal of that would be. Who cares that there's a dozen or so unused deprecated classes? We need to treat each API individually, so the questions are more of the "Can we delete X" kind..